Library/Homebrew/tab.rb

```diff
@@ -242,6 +242,26 @@ def compiler
     super || DevelopmentTools.default_compiler
   end
 
+  def homebrew_tag
+    homebrew_version.sub(/\-\d+\-g([a-f0-9]+)(?:\-dirty)?\Z/, "")
+  end
+
+  def parsed_homebrew_version
+    return Version::NULL if homebrew_version.nil?
+    Version.new(homebrew_tag)
+  end
+
+  # Whether there is reliable runtime dependency information in the receipt.
+  def reliable_runtime_dependencies?
```

Comment:
This would have a side-effect of removing the incorrect lists if the tab was ever modified. I don't foresee that being an issue though, since they're wrong. What do you think?